### DISCUSSION TITLE
有没有提交图标库的 翻遍整个antv也没有找到 还是说 需要自己实现

### DISCUSSION THREAD

**Discussion by 2010363329** [antvis/antvis.github.io#165] [Q&A] [answered]
<img width="739" alt="image" src="https://github.com/antvis/antvis.github.io/assets/76477581/f95d560e-4fdf-44e2-90e9-e95f65ab6d67">
想要这种图标 不知道有没有  还是说没有

**Comment by pearmini** [comment, marked as answer]
有的，可以看看 G2 色块图 + 分面例子。https://g2.antv.antgroup.com/examples/composition/space/#space-flex
